src/ripple/app/ledger/tests/DeferredCredits.test.cpp

```diff
@@ -55,228 +56,198 @@ class DeferredCredits_test : public beast::unit_test::suite
     {
         testcase ("selfFunding");
 
-        auto const keyType = KeyType::ed25519;
-        std::uint64_t const xrp = std::mega::num;
-
-        auto master = createAccount ("masterpassphrase", keyType);
-
-        std::shared_ptr<Ledger const> LCL;
-        Ledger::pointer ledger;
-        std::tie (LCL, ledger) = createGenesisLedger (100000 * xrp, master);
-
-        auto accounts =
-            createAndFundAccountsWithFlags (master,
-                                            {"snd", "rcv", "gw1", "gw2"},
-                                            keyType,
-                                            10000 * xrp,
-                                            ledger,
-                                            LCL,
-                                            asfDefaultRipple);
-        auto& gw1 = accounts["gw1"];
-        auto& gw2 = accounts["gw2"];
-        auto& snd = accounts["snd"];
-        auto& rcv = accounts["rcv"];
+        using namespace jtx;
+        Env env (*this);
+        Account const gw1 ("gw1");
+        Account const gw2 ("gw2");
+        Account const snd ("snd");
+        Account const rcv ("rcv");
```

Comment:
All members of `Account` are const. Putting `const` on the local doesn't hurt. But it also doesn't help.